azkaban-common/src/main/java/azkaban/executor/container/ContainerizedWatch.java

```diff
@@ -0,0 +1,18 @@
+package azkaban.executor.container;
+
+public interface ContainerizedWatch {
```

Comment:
Please add javadoc comment for this interface.

Reply:
Done.